src/sentry/replays.py

```diff
@@ -1,10 +1,6 @@
 from __future__ import absolute_import
-
-import socket
-
-from httplib import HTTPConnection, HTTPSConnection
-from urllib import urlencode
-from urlparse import urlparse
+from requests import RequestException
```

Comment:
Oh, I guess it gets imported into the root package too: https://github.com/kennethreitz/requests/blob/master/requests/__init__.py#L63-L66